hoomd/md/pytest/test_thermo.py

```diff
@@ -155,11 +158,11 @@ def test_system_rotational_dof(simulation_factory, device):
     sim.operations.add(thermo)
 
     integrator = hoomd.md.Integrator(dt=0.0001, integrate_rotational_dof=True)
-    integrator.aniso = True
-    integrator.methods.append(hoomd.md.methods.NVT(filt, tau=1, kT=1))
+    thermostat = hoomd.md.methods.thermostats.MTTK(kT=1.0, tau=1.0)
+    integrator.methods.append(hoomd.md.methods.ConstantVolume(filt, thermostat))
     sim.operations.integrator = integrator
 
-    sim.run(1)
+    sim.run(0)
```

Comment:
Just to make sure I understand: sim.run(1) was changed to sim.run(0) because the thermostat should update velocities even for sim.run(0), correct?

Reply:
With `sim.run(1)`, this test ran one timestep of integration which modifies the velocities slightly from their set point. Therefore the later assert was inaccurate because it was checking the thermodynamic properties resulting from the set velocities against modified velocities.

`sim.run(0)` is necessary so that the C++ implementation of `ComputeThermo` is instantiated and available to compute the properties. With `sim.run(0)`, accessing e.g. `thermo.kinetic_temperature` would result in an error.